Automattic/jetpack
# Social : Update connections management links for classic editor

## Proposed changes:

* Update connections management link for classic editor

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to classic editor on a Jetpack site
* Confirm that the settings link points to Social admin or Jetpack sharing sections page
* Do the same for simple and atomic sites
* Confirm that the settings link points to Calypso


<img width="277" alt="image" src="https://github.com/Automattic/jetpack/assets/18226415/7e4ff52e-ea5f-44eb-bfa7-b1b098d6562e">
